inst/css/BS4/pkgdown.css

```diff
@@ -298,40 +299,23 @@ nav[data-toggle="toc"] .nav .nav > li > a {
 /* h1 */
 
 h1.display-4 {
-  font-size: 2.5rem;
-}
-
-/* Links color contrast */
-
-a {
-  color: #0054AD;
+  font-size: $h1-font-size;
 }
 
-a:hover {
-  color: #0054AD;
-}
-
-
 /* navbar active link */
 .navbar-light .navbar-nav .active > .nav-link {
-   background: #e9ecef;
+   background: $secondary;
    border-radius: 1rem;
  }
 .navbar-nav .nav-link {
  padding-left: 0.5rem;
  }
 
-@include media-breakpoint-up(md) {
-  .navbar-nav .nav-link {
- padding-left: 0rem;
- }
-}
-
 
 /* Footnotes --------------------------------------------- */
 
 .popover {
-  max-width: min(100vw, 32rem);
+  max-width: Min(100vw, 32rem);
```

Comment:
Why doesn't `min()` work here? 

> If a min() or max() function call is valid plain CSS, it will be compiled to a CSS min() or max() call. 
> — https://sass-lang.com/documentation/syntax/special-functions#min-and-max

Reply:
Because libsass doesn't support it :crying_cat_face: 

![image](https://user-images.githubusercontent.com/8360597/118495843-dfb74380-b723-11eb-8b6f-500ba5c1af3c.png)